recipe/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# *****************************************************************
+# (C) Copyright IBM Corp. 2022. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *****************************************************************
+
+autoconf
+
+CFLAGS_INIT="-march=z9-109" LD=$CC ./configure --prefix="${PREFIX}"
```

Comment:
Are we using march=z9-109 instead of march=z14 because of trying to support older z versions ?

Reply:
`CFLAGS_INIT` is only used to build `zdnn_init.c`. main functionality of the library is built with `CFLAGS` which is using `-march=z14`.